.eslintrc.js

```diff
@@ -9,6 +9,7 @@ module.exports = {
     "browser": true
   },
   "rules": {
-    "react/jsx-filename-extension": 0
+    "react/jsx-filename-extension": 0,
+    "no-plusplus": 0,
```

Comment:
Hmm ok, useful to know, did not know the reason for this rule.

Have removed it again, but not removed usage of `++`/`--` in the codebase, can remove them as we progress.